notifications/notifications.go

```diff
@@ -0,0 +1,93 @@
+package notifications
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/pkg/errors"
+)
+
+// Sender contains the configuration information to send events to the notification service
+type Sender struct {
+	URL string
+}
+
+// EmailMessage contains the required fields for formatting email messages
+type EmailMessage struct {
+	Subject string `json:"subject"`
+	Body    string `json:"body"`
+}
+
+// SlackMessage contains the required fields for formatting slack messages
+type SlackMessage struct {
+	Text string `json:"text"`
```

Comment:
should we add `"username": "Weave Cloud"`?

Reply:
I think the notification service itself should decide on what the username should be. This will be appearing in customer slack messages/emails, so it should be something "on-brand".

Reply:
https://github.com/weaveworks/notification/issues/43